manifests/cluster-manager/hub/cluster-manager-grpc-server-service.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ .ClusterManagerName }}-grpc-server
+  namespace: {{ .ClusterManagerNamespace }}
+  labels:
+    {{ if gt (len .Labels) 0 }}
+    {{ range $key, $value := .Labels }}
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)